app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -30,6 +42,30 @@ def supports_port?
     true
   end
 
+  def supported_api_versions
+    supported_api_versions_from_cache
+  end
+
+  def supported_api_versions_from_cache
+    Cacher.new(cache_key).fetch_fresh(last_refresh_date) { supported_api_verions_from_sdk }
+  end
+
+  def cache_key
+    "REDHAT_EMS_CACHE_KEY_#{id}"
+  end
+
+  def supported_api_verions_from_sdk
+    username = authentication_userid(:basic)
+    password = authentication_password(:basic)
+    probe_args = { :hostname => hostname, :port => port, :username => username, :password => password }
+    probe_results = OvirtSDK4::Probe.probe(probe_args)
```

Comment:
So, this makes a live connection to the ovirt server, right?
What happens if the server is unreachable?

I'm also not sure about this, because we create a matrix of supported features https://github.com/ManageIQ/manageiq/pull/11281